framework-cloud/framework-openshift/src/main/java/org/kie/cloud/openshift/settings/builder/WorkbenchSettingsBuilderImpl.java

```diff
@@ -0,0 +1,80 @@
+/*
+ * Copyright 2017 JBoss by Red Hat.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package org.kie.cloud.openshift.settings.builder;
+
+import java.util.HashMap;
+import java.util.Map;
+import org.kie.cloud.api.deployment.constants.DeploymentConstants;
+import org.kie.cloud.api.settings.DeploymentSettings;
+import org.kie.cloud.api.settings.builder.WorkbenchSettingsBuilder;
+import org.kie.cloud.openshift.constants.OpenShiftConstants;
+import org.kie.cloud.openshift.constants.OpenShiftTemplateConstants;
+import org.kie.cloud.openshift.settings.DeploymentSettingsImpl;
+
+public class WorkbenchSettingsBuilderImpl implements WorkbenchSettingsBuilder {
+
+    private Map<String, String> envVariables;
+    private final String appTemplate = OpenShiftConstants.getKieAppTemplateWorkbench();
+
+    public WorkbenchSettingsBuilderImpl() {
+        envVariables = new HashMap<>();
+
+        envVariables.put(OpenShiftTemplateConstants.KIE_ADMIN_USER, DeploymentConstants.getWorkbenchUser());
+        envVariables.put(OpenShiftTemplateConstants.KIE_ADMIN_PWD, DeploymentConstants.getWorkbenchPassword());
+        // By default use Workbench as maven repo, repo URL is derived from Workbench automatically if not defined
+        envVariables.put(OpenShiftTemplateConstants.MAVEN_REPO_USERNAME, DeploymentConstants.getWorkbenchUser());
```

Comment:
Maven repo user should be configured for Kie server as it needs the value (for now as Workbench doesn't have capability to connect itself to external Maven repo).